docs/intro/quickstart/access.rst

```diff
@@ -0,0 +1,423 @@
+.. _ref_quickstart_access:
```

Comment:
How about making this the last section? I'm thinking so as this one gets a bit more difficult and there's a lot going one. Maybe we can cover everything else before access policies, so that we don't scare people away

Reply:
Yeah, I'm fine with that!

Reply:
Another thought - maybe this can be a part of another auth-focused guide?

Reply:
As discussed, we're going to remove the access control/policies stuff from the initial quickstart and just save most of that for the auth extension tutorial that will build on this.